### PR TITLE
Revise Floodgate overview page

### PR DESCRIPTION
**Pull request by mcchampions**
Revise content in wiki/floodgate/index.md
and fix wiki/floodgate no showing the sitebar

### CODE REVIEW

File: wiki/floodgate/index.md [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
 
-# Floodgate
-## What is Floodgate? {#what-is-floodgate}
+# Overview
 Floodgate is a hybrid mode plugin that allows Minecraft: Bedrock Accounts to join Minecraft: Java Edition servers without needing a Minecraft: Java Edition account. 
 This is something you install in addition to Geyser. Unlike Geyser, Floodgate can only be installed as a plugin on Spigot (including Paper and forks), Bungeecord, Velocity, and as a mod on Fabric servers, and can only be utilized on servers that have it installed.
 
-## What has Floodgate 2.0 to offer? {#what-has-floodgate-20-to-offer}
-Floodgate 2.0 is a rewrite of Floodgate 1.0, with various bugfixes and improvements. It also introduces new features: being able to [see Bedrock player skins on Java edition](/wiki/floodgate/features#what-is-skin-uploading),
+Now, the version of Floodgate is 2.0. Floodgate 2.0 is a rewrite of Floodgate 1.0, with various bugfixes and improvements. It also introduces new features: being able to [see Bedrock player skins on Java edition](/wiki/floodgate/features#what-is-skin-uploading),
```
Comment (onebeastchris) [inline wiki/floodgate/index.md:11]: You can drop the mention/comparison of floodgate entirely now, it's been out for a while now

**Review by onebeastchris — CHANGES_REQUESTED** [review]

File: wiki/floodgate/index.md [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
 description: 'Floodgate is a hybrid mode plugin that allows Minecraft: Bedrock Accounts to join Minecraft: Java Edition servers without needing a Minecraft: Java Edition account.'
 ---
 
-# Floodgate
-## What is Floodgate? {#what-is-floodgate}
-Floodgate is a hybrid mode plugin that allows Minecraft: Bedrock Accounts to join Minecraft: Java Edition servers without needing a Minecraft: Java Edition account. 
-This is something you install in addition to Geyser. Unlike Geyser, Floodgate can only be installed as a plugin on Spigot (including Paper and forks), Bungeecord, Velocity, and as a mod on Fabric servers, and can only be utilized on servers that have it installed.
+# Overview
+Floodgate is a hybrid mode plugin. It introduces new features:
+- being able to join Minecraft: Java Edition servers with Minecraft: Bedrock Accounts without needing a Minecraft: Java Edition account.
```
Comment (onebeastchris) [inline wiki/floodgate/index.md:9]:
```suggestion
- Ability to join Minecraft: Java Edition servers with Minecraft: Bedrock accounts without needing a paid Minecraft: Java Edition account. 
```

File: wiki/floodgate/index.md [hunk 3]
```diff
@@ -1,15 +1,14 @@
...
 ---
 
-# Floodgate
-## What is Floodgate? {#what-is-floodgate}
-Floodgate is a hybrid mode plugin that allows Minecraft: Bedrock Accounts to join Minecraft: Java Edition servers without needing a Minecraft: Java Edition account. 
-This is something you install in addition to Geyser. Unlike Geyser, Floodgate can only be installed as a plugin on Spigot (including Paper and forks), Bungeecord, Velocity, and as a mod on Fabric servers, and can only be utilized on servers that have it installed.
+# Overview
+Floodgate is a hybrid mode plugin. It introduces new features:
+- being able to join Minecraft: Java Edition servers with Minecraft: Bedrock Accounts without needing a Minecraft: Java Edition account. 
+- being able to [see Bedrock player skins on Java edition](/wiki/floodgate/features#what-is-skin-uploading).
```
Comment (onebeastchris) [inline wiki/floodgate/index.md:10]:
```suggestion
- Ability to [see Bedrock player skins on Java edition](/wiki/floodgate/features#what-is-skin-uploading).
```

File: wiki/floodgate/index.md [hunk 4]
```diff
@@ -1,15 +1,14 @@
...
+- being able to [use Bedrock forms](/wiki/geyser/forms/).
```
Comment (onebeastchris) [inline wiki/floodgate/index.md:11]:
```suggestion
- Plugins being able to [send Bedrock forms](/wiki/geyser/forms/).
```

File: wiki/floodgate/index.md [hunk 5]
```diff
@@ -1,15 +1,14 @@
...
+- being able to [enable Global Api](/wiki/api/api.geysermc.org/global-api/)(including enabling [Global Linking](/wiki/floodgate/linking#what-is-global-linking) and other features).
```
Comment (onebeastchris) [inline wiki/floodgate/index.md:12]:
```suggestion
- Ability to query players through the [Global Api](/wiki/api/api.geysermc.org/global-api/), 
- Allowing Bedrock players to link to a Java account using [Global Linking](/wiki/floodgate/linking#what-is-global-linking), or Local Linking.
```

**Review by onebeastchris — CHANGES_REQUESTED** [review]

File: wiki/floodgate/index.md (hunk 4 above)
Comment (Chew) [reply wiki/floodgate/index.md:11]: `Ability for plugins to` I think would fit better?

**Review by onebeastchris — APPROVED** [review]

**Pull request merged**